src/ripple/test/jtx/impl/Env.cpp

```diff
@@ -91,7 +92,12 @@ Env::Env (beast::unit_test::suite& test_)
     Pathfinder::initPathTable();
 }
 
-std::shared_ptr<ReadView const>
+Env::Env(beast::unit_test::suite& test_)
+    : Env(test_, std::move(makeConfig()))
```

Comment:
@vinniefalco I added a ctor override to `Env` so I could pass in a custom `Config`, so I could take advantage of the objects constructed and owned by the `Application` with my own settings.  The alternative was to instantiate another copy of *all* the relevant objects in my test, then risk that something in `Env` would its internal version and cause havok.

@seelabs Fixed. Weird. I could have sworn VS gave me a build error without it, but nope.